### DISCUSSION TITLE
SECTION 3: REMIX IDE

### DISCUSSION THREAD

**Discussion by Neke-Chukwu** [Cyfrin/foundry-full-course-cu#1917] [Q&A] [answered]
why is my remix ide code not deploying? its saying "evm version: paris"

**Comment by EngrPips** [comment, marked as answer]
Hello @Hack-Mantis, You can change the `evm version` if that is required to deploy your contract.

**Reply by Neke-Chukwu** [reply]
Thanks, that worked

**Reply by EngrPips** [reply]
You are welcome, buddy. Keep crushing it.
